modules/ztp-configuring-cluster-policies.adoc

```diff
@@ -4,11 +4,11 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="ztp-configuring-cluster-policies_{context}"]
-= Configuring managed clusters with policies and PolicyGenTemplate resources
+= Configuring managed clusters with policies and `{policy-gen-cr}` resources
```

Comment:
```suggestion
= Configuring managed clusters with policies and {policy-gen-cr} resources
```
From [OCP docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings): "Do not use backticks or other markup in assembly or module headings."